### ISSUE TITLE
Link color contrast on dark backgroud

### ISSUE THREAD

**Issue by mpaiva** [mpaiva/CRDC-DSRC#5]
### SC 1.4.3: Contrast (Minimum) (Level AA)
The visual presentation of [text](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum#dfn-text) and [images of text](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum#dfn-image-of-text) has a [contrast ratio](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum#dfn-contrast-ratio) of at least 4.5:1, except for the following:

- Large Text - [Large-scale](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum#dfn-large-scale) text and images of large-scale text have a contrast ratio of at least 3:1;
- Incidental - Text or images of text that are part of an inactive [user interface component](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum#dfn-user-interface-component), that are [pure decoration](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum#dfn-pure-decoration), that are not visible to anyone, or that are part of a picture that contains significant other visual content, have no contrast requirement.
- Logotypes - Text that is part of a logo or brand name has no contrast requirement.

NOTE: Even though this is part of the logo, it is still perceived as a text link by the low-vision user.

## How to fix:
Change color links to `#008bef` for text that are on **dark** background

```
.usa-navbar a:hover {
    color: #008bef;
}
```

### Before:
![image](https://github.com/mpaiva/CRDC-DSRC/assets/781670/ea9ff8b1-eefb-4ff5-838e-5793ad704ebb)

### After:
![image](https://github.com/mpaiva/CRDC-DSRC/assets/781670/0b0c8af6-8639-487c-a9bc-d4da14453dbf)

![image](https://github.com/mpaiva/CRDC-DSRC/assets/781670/8f0ff878-a249-4940-91d1-20ed2d462de6)
